Make check errors

```
~/Development/src/github.com/appcelerator/amp(master ✔) make check
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:7:6: exported type Details should have comment or be unexported
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:17:6: exported type VersionConfig should have comment or be unexported
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:17:6: type name will be used as version.VersionConfig by other packages, and that stutters; consider calling this Config
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:22:6: exported type Server should have comment or be unexported
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:34:1: comment on exported method Server.List should be of the form "List ..."
/go/src/github.com/appcelerator/amp/api/rpc/version/version.go:49:1: comment on exported method VersionConfig.AmplifierOK should be of the form "AmplifierOK ..."
/go/src/github.com/appcelerator/amp/cmd/adm-server/servercore/amp-service.go:417:13: if blockends with a return statement, so drop this else and outdent its block
/go/src/github.com/appcelerator/amp/cmd/adm-server/servercore/amp-service.go:500:10: if blockends with a return statement, so drop this else and outdent its block
/go/src/github.com/appcelerator/amp/cmd/amp/storage.go:104:21: error strings should not be capitalized or end with punctuation or a newline
/go/src/github.com/appcelerator/amp/cmd/amp/storage.go:129:21: error strings should not be capitalized or end with punctuation or a newline
/go/src/github.com/appcelerator/amp/cmd/amp/storage.go:154:21: error strings should not be capitalized or end with punctuation or a newline
/go/src/github.com/appcelerator/amp/cmd/amp/version.go:70:10: if block ends with a return statement, so drop this else and outdent its block
/go/src/github.com/appcelerator/amp/cmd/amp-function-listener/main.go:100:1: exported function Index should have comment or be unexported
/go/src/github.com/appcelerator/amp/cmd/ampadm/bootstrap-docker.go:19:5: exported var DockerChannels should have comment or be unexported
/go/src/github.com/appcelerator/amp/cmd/ampadm/bootstrap-docker.go:26:2: const dockerUrl should be dockerURL
/go/src/github.com/appcelerator/amp/cmd/ampadm/bootstrap-docker.go:27:2: const dockerGetUrl should be dockerGetURL
/go/src/github.com/appcelerator/amp/cmd/ampadm/client.go:21:2: exported const ClearScreen should have comment (or a comment on this block) or be unexported
/go/src/github.com/appcelerator/amp/pkg/nats-streaming/nats-streaming.go:11:1: comment on exported type NatsStreaming should be of the form "NatsStreaming ..." (with optional leading article)
/go/src/github.com/appcelerator/amp/pkg/nats-streaming/nats-streaming.go:17:46: method parameter clusterId should be clusterID
/go/src/github.com/appcelerator/amp/pkg/nats-streaming/nats-streaming.go:17:64: method parameter clientId should be clientID
/go/src/github.com/appcelerator/amp/pkg/nats-streaming/nats-streaming.go:32:1: exported method NatsStreaming.GetClient should have comment or be unexported
/go/src/github.com/appcelerator/amp/pkg/nats-streaming/nats-streaming.go:36:1: exported method NatsStreaming.Close should have comment or be unexported
```

--- a/api/rpc/version/version.go
+++ b/api/rpc/version/version.go
@@ -4,6 +4,7 @@ import (
 	"golang.org/x/net/context"
 )
 
+//Details detailled information
 type Details struct {
 	Version    string
 	Build      string
@@ -14,11 +15,13 @@ type Details struct {
 	Arch       string
 }
 
-type VersionConfig struct {
+// Config version information
+type Config struct {
 	AMP       *Details
 	Amplifier *Details
 }
 
+// Server server information
 type Server struct {
 	Version   string
 	Port      string
@@ -31,7 +34,7 @@ func init() {
 	// no setup
 }
 
-// Returns Amplifier info from Server.go
+// List Returns Amplifier info from Server.go
 func (s *Server) List(ctx context.Context, in *ListRequest) (*ListReply, error) {
 
 	response := &ListReply{
@@ -46,7 +49,7 @@ func (s *Server) List(ctx context.Context, in *ListRequest) (*ListReply, error) 
 	return response, nil
 }
 
-// Checks if AMP is connected to Amplifier
-func (v VersionConfig) AmplifierOK() bool {
+// AmplifierOK Checks if AMP is connected to Amplifier
+func (v Config) AmplifierOK() bool {
 	return v.Amplifier != nil
 }

--- a/cmd/adm-server/servercore/amp-service.go
+++ b/cmd/adm-server/servercore/amp-service.go
@@ -414,9 +414,8 @@ func (s *AMPInfraManager) updateServiceStates(service *ampService) error {
 					if s.Force {
 						s.forceService(service)
 						return nil
-					} else {
-						return fmt.Errorf("Service %s startup timeout", service.name)
 					}
+					return fmt.Errorf("Service %s startup timeout", service.name)
 				}
 			}
 			service.ready = false
@@ -497,10 +496,9 @@ func (s *AMPInfraManager) createService(service *ampService) error {
 			s.printf(colWarn, "Service %s image %s doesn't exist", service.name, service.image)
 			s.forceService(service)
 			return nil
-		} else {
-			s.printf(colError, "Service %s image %s doesn't exist", service.name, service.image)
-			return fmt.Errorf("Service image error")
 		}
+		s.printf(colError, "Service %s image %s doesn't exist", service.name, service.image)
+		return fmt.Errorf("Service image error")
 	}
 	r, err := s.docker.ServiceCreate(s.ctx, *service.spec, options)
 	if err != nil {

--- a/cmd/amp-function-listener/main.go
+++ b/cmd/amp-function-listener/main.go
@@ -97,6 +97,7 @@ func main() {
 	log.Fatal(http.ListenAndServe(listenAddr, router))
 }
 
+// Index index
 func Index(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 	// Get the function id or name from the URL
 	idOrName := strings.TrimSpace(p.ByName("function"))

--- a/cmd/amp/storage.go
+++ b/cmd/amp/storage.go
@@ -101,7 +101,7 @@ func storagePut(amp *client.AMP, cmd *cobra.Command, args []string) error {
 //by invoking the corresponding rpc/storage method
 func storageGet(amp *client.AMP, cmd *cobra.Command, args []string) error {
 	if len(args) > 1 {
-		return errors.New("too many arguments - check again!")
+		return errors.New("too many arguments - check again")
 	} else if len(args) == 0 {
 		return errors.New("must specify storage key")
 	}
@@ -126,7 +126,7 @@ func storageGet(amp *client.AMP, cmd *cobra.Command, args []string) error {
 // by invoking the corresponding rpc/storage method
 func storageDelete(amp *client.AMP, cmd *cobra.Command, args []string) error {
 	if len(args) > 1 {
-		return errors.New("too many arguments - check again!")
+		return errors.New("too many arguments - check again")
 	} else if len(args) == 0 {
 		return errors.New("must specify storage key")
 	}
@@ -151,7 +151,7 @@ func storageDelete(amp *client.AMP, cmd *cobra.Command, args []string) error {
 // key-value pairs by invoking the corresponding rpc/storage method
 func storageList(amp *client.AMP, cmd *cobra.Command, args []string) error {
 	if len(args) > 0 {
-		return errors.New("too many arguments - check again!")
+		return errors.New("too many arguments - check again")
 	}
 	request := &storage.ListStorage{}
 	client := storage.NewStorageClient(amp.Conn)

--- a/cmd/amp/version.go
+++ b/cmd/amp/version.go
@@ -50,7 +50,7 @@ func list(amp *client.AMP, cmd *cobra.Command, args []string) error {
 	}
 	var doc bytes.Buffer
 
-	vd := version.VersionConfig{
+	vd := version.Config{
 		AMP: &version.Details{
 			Version:    Version,
 			Build:      Build,
@@ -67,14 +67,13 @@ func list(amp *client.AMP, cmd *cobra.Command, args []string) error {
 		reply, err := client.List(context.Background(), request)
 		if err != nil {
 			return err
-		} else {
-			vd.Amplifier = &version.Details{
-				Version:   reply.Reply.Version,
-				Port:      reply.Reply.Port,
-				GoVersion: reply.Reply.Goversion,
-				Os:        reply.Reply.Os,
-				Arch:      reply.Reply.Arch,
-			}
+		}
+		vd.Amplifier = &version.Details{
+			Version:   reply.Reply.Version,
+			Port:      reply.Reply.Port,
+			GoVersion: reply.Reply.Goversion,
+			Os:        reply.Reply.Os,
+			Arch:      reply.Reply.Arch,
 		}
 	}
 

--- a/cmd/ampadm/bootstrap-docker.go
+++ b/cmd/ampadm/bootstrap-docker.go
@@ -16,15 +16,11 @@ import (
 	"strings"
 )
 
-var DockerChannels = [...]string{"main"}
-
-//var DockerChannels = [...]string{"main", "testing"}
-
 const (
 	minimumDockerEngineVersion = "1.12.3"
 	dockerAPIMinVersion        = "1.24"
-	dockerUrl                  = "unix:///var/run/docker.sock"
-	dockerGetUrl               = "http://get.docker.com"
+	dockerURL                  = "unix:///var/run/docker.sock"
+	dockerGetURL               = "http://get.docker.com"
 	dockerRemoteAPIPort        = "2375"
 )
 
@@ -35,7 +31,7 @@ func (c *clusterClient) getDockerServerVersion() (string, error) {
 	// first try the Docker api
 	if c.dockerClient == nil {
 		defaultHeaders := map[string]string{"User-Agent": "amp-bootstrap"}
-		c.dockerClient, _ = dockerclient.NewClient(dockerUrl, dockerAPIMinVersion, nil, defaultHeaders)
+		c.dockerClient, _ = dockerclient.NewClient(dockerURL, dockerAPIMinVersion, nil, defaultHeaders)
 	}
 	if c.dockerClient != nil {
 		version, err := c.dockerClient.ServerVersion(c.ctx)
@@ -165,7 +161,7 @@ func (c *clusterClient) startDockerEngine() error {
 
 // Use the official script to install Docker
 func (c *clusterClient) installDockerEngine() error {
-	resp, err := http.Get(dockerGetUrl)
+	resp, err := http.Get(dockerGetURL)
 	if err != nil {
 		return err
 	}
@@ -239,7 +235,7 @@ func (c *clusterClient) isSwarmInit() bool {
 func (c *clusterClient) joinSwarm() error {
 	if c.dockerClient == nil {
 		defaultHeaders := map[string]string{"User-Agent": "amp-bootstrap"}
-		dclient, err := dockerclient.NewClient(dockerUrl, dockerAPIMinVersion, nil, defaultHeaders)
+		dclient, err := dockerclient.NewClient(dockerURL, dockerAPIMinVersion, nil, defaultHeaders)
 		if err != nil {
 			return err
 		}

--- a/cmd/ampadm/client.go
+++ b/cmd/ampadm/client.go
@@ -18,7 +18,9 @@ import (
 )
 
 const (
-	ClearScreen    = "\033[2J\033[0;0H"
+	//ClearScreen ANSI Escape code
+	ClearScreen = "\033[2J\033[0;0H"
+	//MoveCursorHome ANSI Escape code
 	MoveCursorHome = "\033[0;0H"
 )
 

--- a/pkg/nats-streaming/nats-streaming.go
+++ b/pkg/nats-streaming/nats-streaming.go
@@ -8,20 +8,20 @@ import (
 	"time"
 )
 
-// NATS-Streaming wrapper
+// NatsStreaming NATS-Streaming wrapper
 type NatsStreaming struct {
 	client stan.Conn
 }
 
 // Connect to NATS-Streaming
-func (ns *NatsStreaming) Connect(url string, clusterId string, clientId string, timeout time.Duration) error {
-	log.Printf("Connecting to nats streaming, url: %s, clusterId: %s, clientId: %s, timeout: %s\n", url, clusterId, clientId, timeout)
+func (ns *NatsStreaming) Connect(url string, clusterID string, clientID string, timeout time.Duration) error {
+	log.Printf("Connecting to nats streaming, url: %s, clusterId: %s, clientId: %s, timeout: %s\n", url, clusterID, clientID, timeout)
 	nc, err := nats.Connect(url, nats.Timeout(timeout))
 	if err != nil {
 		return fmt.Errorf("unable to connect to nats streaming: %v", err)
 	}
 
-	ns.client, err = stan.Connect(clusterId, clientId, stan.NatsConn(nc), stan.ConnectWait(timeout))
+	ns.client, err = stan.Connect(clusterID, clientID, stan.NatsConn(nc), stan.ConnectWait(timeout))
 	if err != nil {
 		return fmt.Errorf("unable to connect to nats streaming: %v", err)
 	}
@@ -29,10 +29,12 @@ func (ns *NatsStreaming) Connect(url string, clusterId string, clientId string, 
 	return nil
 }
 
+// GetClient return client
 func (ns *NatsStreaming) GetClient() stan.Conn {
 	return ns.client
 }
 
+// Close the client
 func (ns *NatsStreaming) Close() error {
 	return ns.client.Close()
 }

